src/ert/gui/simulation/run_dialog.py

```diff
@@ -532,6 +532,21 @@ def restart_failed_realizations(self) -> None:
             self.run_experiment(restart=True)
 
 
+class CopyDebugInfoButton(QPushButton):
+    def __init__(self, on_click: Callable[[], None]):
+        QPushButton.__init__(self, "Copy Debug Info")
+        self.setToolTip("Copies useful information to clipboard")
+        self.setObjectName("copy_debug_info_button")
+        self.setFixedWidth(140)
+
+        def alternate_button_text_on_click_and_call_callback() -> None:
+            self.setText("Copied...")
```

Comment:
Consider alternatives like;
`Copied!`
`Copied to clipboard!`
`Copied to clipboard`

Reply:
I tried the alternatives, but I think Copied... looks better